core/testdata/proto/test.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package test;
+
+service TestService {
+  rpc Test (TestRequest) returns (TestReply) {}
+}
+
+message TestRequest {
+  string field1 = 1;
+}
+
+message TestReply {
+  string field1 = 1;
+}
```

Comment:
Also, why is this file being added? It does not appear to be needed to rename the repo.

Reply:
Yup agreed, it was added to ensure that `make test` would run successfully